tests/test_pgbouncer.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+import subprocess
+from tests.common import load_check
+
+from nose.plugins.attrib import attr
+
+import time
+from pprint import pprint
+
+@attr(requires='pgbouncer')
+class TestPgbouncer(unittest.TestCase):
```

Comment:
That would be nice to use the new test interface, and much simpler to write...
If you don't have time to do it, tell me and I'll take a pass at it I think before merging the PR.